File: python/ex11.py
# 11. Soma de cada linha
# Dada uma matriz, calcule a soma dos elementos de cada linha e exiba os
# resultados.

from utils import matriz

soma = 0

for i, rows in enumerate(matriz):
    for num in rows:
        soma+=num
    print(f"Soma linha {i + 1}: {soma}")
    soma = 0

File: javascript/ex11.js
/*

11. Soma de cada linha
Dada uma matriz, calcule a soma dos elementos de cada linha e exiba os
resultados.

*/

const matriz = require("../matriz.json")

soma = 0

matriz.forEach((row, i) => {
    row.forEach(num => {
        soma += num
    })
    console.log(`Soma da linha ${i+1}: ${soma}`)
    soma = 0
})